src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -75,7 +75,7 @@ public ConcurrentBag(IEnumerable<T> collection)
         /// <see cref="ConcurrentBag{T}"/>. The value can be a null reference
         /// (Nothing in Visual Basic) for reference types.</param>
         public void Add(T item) =>
-            GetCurrentThreadWorkStealingQueue(forceCreate: true)
```

Comment:
presumably item is allowed to be null, should we add TODO?

Reply:
similar comment to other similar APIs in this file

Reply:
It's allowed to be null if `T` is nullable, in which case the caller will see whatever nullable T they provided.  What would need to change here?

Reply:
I guess that the nullability will be expressed based on the type of T when used.

Reply:
what I meant is even if T is marked as non-nullable this will still be nullable

Reply:
> what I meant is even if T is marked as non-nullable this will still be nullable

If T is non-nullable, e.g. `string` instead of `string?`, then this will warn if you try to pass in a null, which is exactly what we want to happen.  What am I missing?

Reply:
right